packages/zero-runtime/src/processors/styled.ts

```diff
@@ -192,8 +202,93 @@ export class StyledProcessor extends BaseProcessor {
     }`;
   }
 
+  private generateArtifacts() {
+    const artifacts: [Rules, Replacements][] = this.collectedStyles.map(([className, cssText]) => {
+      const rules: Rules = {
+        [`.${className}`]: {
+          className,
+          cssText,
+          displayName: this.displayName,
+          start: this.location?.start ?? null,
+        },
+      };
+      // @TODO - Refactor for finer location tracking in original code.
```

Comment:
Sorry, what's this comment means?

Reply:
So that when you click on the variants classname, we can take you to the specific variant definition in the code instead of just taking you to the `styled` call.

Reply:
It's not super important right now.